package.json

```diff
@@ -121,6 +121,9 @@
     "*": {
       "*": [
         "types/*"
+      ],
+      "cjs/src/*": [
+        "types/*"
```

Comment:
I am not sure I understand what this change does. Do we ever import path like `multiformats/cjs/src/...` because that is what I think it would catch here.

If this is about fixing

```js
import { CID } from 'multiformats'
```

I think suggestion above is probably adding following to `package.json` is a better way to go:


```json
{
  "types": "./types/index.d.ts"
}
```

Reply:
comment below https://github.com/multiformats/js-multiformats/pull/66#issuecomment-809950708

this is for when we publish a `"main"` pointing to cjs/src/index.js, that mapping causes problems for consumers of this package